patch/copy_op.go

```diff
@@ -0,0 +1,15 @@
+package patch
+
+type QCopyOp struct {
```

Comment:
Missing comments here and below (this and other files) for the exported stuff.

Reply:
None of the upstream code uses comments on these structs. I could put one in, if it makes you happy, but what should it say: "QCopyOp holds the parameters for a copy operation"?

Isn't that just useless clutter?

Reply:
Ugh... this is not what the Go convention tells... I'm fine with leaving them out for the sake of consistency with the rest of the repo.